content/docs/user-guide/dvc-files-and-directories.md

```diff
@@ -226,6 +226,35 @@ directory tracked by DVC. Specifically: `md5`, `etag`, or `checksum` (same as in
 Full <abbr>parameters</abbr> (key and value) are listed separately under
 `params`, grouped by parameters file.
 
+### Renaming stage outputs
+
+Let's take the following scenario and rename stage <abbr>output</abbr>
+`keras.h5` to `model.h5` for the `train` stage:
+
+```yaml
```

Comment:
@jorgeorpinel I have modified this based on your inputs and added to this doc as well for comparison. Which doc do you think is better for this section? (I'll remove from the other one). I thought it will look odd in `dvc-files` doc since it only contains explanation and as `dvc move` is related to renaming and this section also is more like a code walkthrough it will be better suited in 'move' doc. Shall I remove it from `move` doc?
___
Also I have removed the `dvc run -f` part since it more complicated. The user will still need to change the name in code and already existing model file (its `.gitignore`  entry) is not deleted.